server/api/v1/coastlines/coastlines.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from sqlalchemy.orm import joinedload
+
+from server import models, schemas
+# TODO Implement Settings to have hardcoded route prefixes for future use
+from server.core.config import Settings
+from server.db.session import get_db
+
+router = APIRouter(prefix="/api/v1/coastlines", tags=["coastlines"])
+
+
+@router.get("/")
```

Comment:
Let's replace this hardcoded string with a variable too. Will be easier to manage all the hardcoded values through a variable in the long run.

Reply:
if you're referring to the prefix url, that is already imported via server.core.config as COASTLINES_V1_STR, and I have placed it in the prefix argument.

Reply:
Nice. I was also referring to the relative root path (`/`).

Reply:
I don't understand what you mean. The relative route path in the coastlines.py file is just /api/v1/coastlines but now without us needing to type out the whole endpoint path because it's in the prefix argument.

Reply:
I meant something like this for consistency:

1. Define the relative root path in another file.
    ```python
    # located in a file with all the other endpoints
    RELATIVE_ROOT = "/"
    ```

2. Use that relative root variable.
    ```python
    # server/api/v1/...
    # ...
    @router.get(RELATIVE_ROOT)
    ```

Reply:
But why though? Isn't this just add extra variables to keep track of?

Reply:
Mainly for consistency, readability and maintainability – we'll discuss this when we chat today!

Reply:
Weird. I tried to make the refactor with constants locally and it didn't work for me. It might not be supported.

Was it the same on your end?

Reply:
@Grieze, let's keep an eye on this potential issue.